src/styles/components/_our-goal-box.scss

```diff
@@ -0,0 +1,46 @@
+@import '../abstracts/_abstracts-dir';
+
+.our-goal-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px $silver-chalice;
+  background-color: transparent;
+
+  &__title {
+    display: none;
+
+    @include respond-above('md') {
+      display: block;
+      padding: rem(32px) 0 rem(16px);
+      margin: 0;
+      font-size: rem(28px);
+      font-weight: bold;
+      line-height: 1.22;
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(36px);
+    }
+  }
+
+  &__description {
+    box-sizing: border-box;
+    padding: rem(16px 28px);
+    margin: auto;
+    font-size: rem(14px);
+    font-weight: 200;
+    text-align: center;
+    line-height: 1.53;
+
+    @include respond-above('md') {
+      height: auto;
+      padding: 0 rem(28px 20px 28px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('lg') {
+      padding: 0 rem(180px 56px 80px);
```

Comment:
I don't think this is a good idea to make this huge amount of right padding (180px). Should we keep the description center like our design in `desktop-1920`?

Reply:
Sure, at first i'm keeping this with design in ``desktop-1440``